tests/metrics/estimate_gas.py

```diff
@@ -183,6 +183,7 @@ def estimate_gas(analyzer: AnalyzeGas = None) -> None:
         compiled_contract = testerchain._raw_contract_cache[contract_name]
 
         version = list(compiled_contract).pop()
+        # FIXME this value includes constructor code size but should not
```

Comment:
What needs to be fixed here?

Reply:
I thought this block worked before but when I checked this time it's not, and not sure is it even possible to get information about contact size inside tx. (I'm getting it from tx itself inside web3). Maybe @cygnusv knows how to fix it